### ISSUE TITLE
Add a translation function

### ISSUE THREAD

**Issue by thomas-illiet** [solstice23/argon-theme#1]
Hello @abc2237512422 

Can you add in your theme a feature to add a translate ?

if this feature is implemented, I would suggest a translation for the French and English language :)

regards
Thomas

**Comment by Waga43** [comment]
Yeah. This would be great. I fell in love with your theme. It is so feature rich. I have had to rẹ-write the entire theme in English since 99% of my target site visitors don't understand Chinese. Now there is a new update, I find it hard to make out time to update my theme to the latest because I am apprehensive of the task of editing it again to English.

By the way, the theme breaks with the latest RankMath SEO plugin. I had to disable my RankMath plugin because I didn't want to change your theme. It is really affecting my SEO. I will open an issue regarding this.

**Comment by solstice23** [comment]
> Yeah. This would be great. I fell in love with your theme. It is so feature rich. I have had to rẹ-write the entire theme in English since 99% of my target site visitors don't understand Chinese. Now there is a new update, I find it hard to make out time to update my theme to the latest because I am apprehensive of the task of editing it again to English.
> 
> By the way, the theme breaks with the latest RankMath SEO plugin. I had to disable my RankMath plugin because I didn't want to change your theme. It is really affecting my SEO. I will open an issue regarding this.

Now English translation is supported, but it has a large amount of text, so only part of the text has been translated. The translation of all texts will be completed in the near future. Some texts are translated by machine, I need some help to translate them better.

**Comment by Waga43** [comment]
Wow!!! Great step in the right direction. Weldone boss. 

After my last post above, I took a bold step and initially updated my theme. It reverted back to the Chinese characters. I had started editing the theme again and had to stop midway. Today I got a prompt that another update is available for my theme and decided to comply. To my utter pleasant surprise, the latest update corrected this and the site is mostly in English now, except for some few areas. Now I don't have to worry too much about editing the code from Chinese to English. You did great. Thanks for sparing your time and efforts to integrate this into the theme. 

Now I can confidently say I have gotten to my last bus stop as regards my WP theme and will have no need to jump from one theme to the other. :) 

Gracias.
